src/containers/GameContainer/saveResultLocal.ts

```diff
@@ -0,0 +1,9 @@
+const SCORE = 'maxScore';
+
+export const saveResultLocal = (score: number): void => {
```

Comment:
🧐
Как вариант. Store в данном случае отсылает к `localStorage`
```suggestion
export const storeResult = (score: number): void => {
```